src/main/java/org/zalando/nakadi/repository/kafka/KafkaPartitionStartStatistics.java

```diff
@@ -0,0 +1,49 @@
+package org.zalando.nakadi.repository.kafka;
+
+import org.zalando.nakadi.domain.NakadiCursor;
+import org.zalando.nakadi.domain.PartitionBaseStatisticsImpl;
+import org.zalando.nakadi.domain.PartitionStartStatistics;
+import org.zalando.nakadi.domain.Timeline;
+
+public class KafkaPartitionStartStatistics extends PartitionBaseStatisticsImpl implements PartitionStartStatistics {
+
+    private final KafkaCursor first;
+
+    public KafkaPartitionStartStatistics(final Timeline timeline, final int partition, final long firstOffset) {
+        super(timeline, KafkaCursor.toNakadiPartition(partition));
+        this.first = new KafkaCursor(timeline.getTopic(), partition, firstOffset);
+    }
+
+    @Override
+    public NakadiCursor getFirst() {
+        // TODO: Support several timelines
+        return first.toNakadiCursor(getTimeline());
+    }
+
+    @Override
+    public NakadiCursor getBeforeFirst() {
+        // TODO: Support several timelines
+        return first.addOffset(-1).toNakadiCursor(getTimeline());
```

Comment:
I just copy/pasted it from here https://github.com/zalando/nakadi/blob/3d551da20b2fa4b5ed99a77a8aad0e120c38e3e6/src/main/java/org/zalando/nakadi/repository/kafka/KafkaPartitionStatistics.java#L34
together with TODO. I believe this problem will be fixed when implementing these TODOs.
But I don't think that this PR is a good place for that.